tests/tests_twobody/test_perturbations.py

```diff
@@ -191,9 +191,8 @@ def test_atmospheric_drag():
 
     # parameters of a body
     C_D = 2.2  # dimentionless (any value would do)
-    A = ((np.pi / 4.0) * (u.m ** 2)).to(u.km ** 2).value  # km^2
-    m = 100  # kg
-    B = C_D * A / m
+    A_over_m = ((np.pi / 4.0) * (u.m ** 2)).to(u.km ** 2).value / 100  # km^2/kg
```

Comment:
These units are missing u.kg -- won't change the test result, but a future reader might be confused.